SimDataFormats/GEMDigiSimLink/interface/ME0DigiSimLink.h

```diff
@@ -0,0 +1,61 @@
+#ifndef ME0OBJECTS_ME0DIGISIMLINK_H
```

Comment:
ME0DigiSimLink is now 
<PRE>
#include "SimDataFormats/GEMDigiSimLink/interface/GEMDigiSimLink.h"
typedef GEMDigiSimLink ME0DigiSimLink;
</PRE>